.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+/ @gomods/contributors @gomods/maintainers
```

Comment:
I was thinking about it after approving, do we want to have maintainers and contributors own everything by default and the only contributors own `/pkg`? Or did you intend to have maintainers own everything and then contributors own `/pkg`?

Reply:
also I think the syntax is `*` for all, not `/`.

Reply:
as well, these rules operate by 'last match precedence'. So without contributors above, the below line will make them owners of `/pkg` over maintainers.